llm-request-plz.el

```diff
@@ -133,7 +153,7 @@ the buffer is turned into JSON and passed to ON-SUCCESS."
               (funcall on-success (json-read-from-string response))))
     :else (lambda (error)
             (when on-error
-              (funcall on-error error)))
+              (llm-request-plz--handle-error error on-error)))
```

Comment:
The fix is appreciated, but it'd be nice to not call into this internal `plz` function, and instead maybe just call `on-error` with an `'error` and a message we create ourselves.

Reply:
What do you mean by internal? The `llm-request-plz--handle-error` function? It's a private function of this module, not plz.el. So I think it's fine here. But feel free to change it as you like.

Reply:
Sorry, my bad - I misread this. This looks good.